tests/cpp.test

```diff
@@ -129,3 +129,21 @@ import Network.Wai.Internal
 #endif
 OUTPUT
 No hints
+
+---------------------------------------------------------------------
+RUN --cpp-simple tests/cpp-file5.hs
```

Comment:
Remove `--cpp-simple`. Otherwise, it runs a simple CPP where lines starting with "#" are simply removed, so it is actually running HLint on 
```
main = 
  print __GLASGOW_HASKELL__
  putStrLn $ show __GLASGOW_HASKELL__
```